app/appsettings.h

```diff
@@ -11,6 +11,7 @@ class AppSettings: public QObject {
     Q_PROPERTY(QString defaultProjectName READ defaultProjectName NOTIFY defaultProjectChanged)
     Q_PROPERTY(QString defaultLayer READ defaultLayer WRITE setDefaultLayer NOTIFY defaultLayerChanged)
     Q_PROPERTY(bool autoCenterMapChecked READ autoCenterMapChecked WRITE setAutoCenterMapChecked NOTIFY autoCenterMapCheckedChanged)
+    Q_PROPERTY(int digitizingPeriod READ digitizingPeriod WRITE setDigitizingPeriod NOTIFY digitizingPeriodChanged)
```

Comment:
Can we rename this to e.g. lineRecordingInterval? So it is clear it only relates to lines and it is consistent with GUI

Reply:
How about VertexRecordingInterval, in case we want to add polygon capturing to Input?